api/app/serializers/project_full_serializer.rb

```diff
@@ -0,0 +1,54 @@
+# Provides a partial serialization of a project model.
+class ProjectFullSerializer < ProjectSerializer
+  include SerializedMetadata
+
+  meta(partial: false)
+
+  attributes :event_count, :metadata, :collections_count, :resources_count,
+             :uncollected_resources_count, :published_text_toc_id, :event_types,
+             :metadata_properties, :citations, :hide_activity, :metadata_formatted,
+             :slug
+
+  link :self do
+    "https://manifold-api.dev/api/v1/projects/#{object.id}"
```

Comment:
This has probably been here forever, but we may as well fix it with our links service.